space_age.py

```diff
@@ -0,0 +1,33 @@
+class SpaceAge(object):
+    __init__(self, seconds):
```

Comment:
@zarniak Ah, sorry! I gave you an incorrect snippet before. 

That line should be:
```
    def  __init__(self, seconds):
```

Classes (defined with `class`) have methods (defined with `def`).